manifests/params.pp

```diff
@@ -4,7 +4,9 @@
 #  currently, only the Redhat family is supported, but this can be easily
 #  extended by changing package names and configuration file paths.
 #
-class haproxy::params {
+class haproxy::params (
+  $sort_options_alphabetic = true,
```

Comment:
You're talking about putting the parameter here to globally modify it? Yeah, passing parameters to `*::params` is not a pattern we use. If we HAVE to have global parameters that apply to all defines, we try to put it in the base class. If that doesn't work for some reason, then we use a `*::globals` class (iirc, postgresql and mongodb do this).

But yeah, moving it somewhere else would be nice.